sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_event.hpp

```diff
@@ -111,6 +116,45 @@ struct _ur_ze_event_list_t {
     }
     return *this;
   }
+
+  // This function allows to merge two _ur_ze_event_lists
+  // The ur_ze_event_list "other" is added to the caller list.
+  // Note that new containers are allocated to contains the additional elements.
+  void fusion(const _ur_ze_event_list_t &other) {
+    if (this != &other) {
+      // save of the previous object values
+      uint32_t preLength = this->Length;
```

Comment:
LLVM coding style is to start variable names with an upper case character - `PreLength`, `PreZeEventList`, `PreUrEventList`, `Other`